internal/middleware/log_api_calls.go

```diff
@@ -18,7 +18,7 @@ func LogApiCalls() Middleware {
 		return func(w http.ResponseWriter, r *http.Request) {
 
 			organisationId := r.Header.Get(config.OrganizationId)
-			aLog := fmt.Sprintf("Organization API: %v called by user: %v", r.URL.Path, token.GetUserName(r))
+			aLog := fmt.Sprintf("%v: %v called by user: %v", r.Method, r.URL.Path, token.GetUserName(r))
```

Comment:
Is this email id? We cannot leak any personal info in the logs.

Reply:
Email id is replaced with user Id.